manifests/istio-control/istio-config/templates/deployment.yaml

```diff
@@ -194,16 +171,6 @@ spec:
       securityContext:
         fsGroup: 1337
       volumes:
-  {{- if or .Values.global.controlPlaneSecurityEnabled (and .Values.global.configValidation (not .Values.global.istiod.enabled)) }}
-      - name: istio-certs
```

Comment:
removing this is causing an error deploying galley manifest

Reply:
can you open an issue with more details? it's very likely we will remove galley entirely by the time we ship 1.6 though, and this commit is not included in 1.5